python/GafferSceneTest/TweakPlugTest.py

```diff
@@ -159,7 +159,7 @@ def testMissingMode( self ) :
 			p.applyTweaks( d, missingMode = GafferScene.TweakPlug.MissingMode.IgnoreOrReplace )
 		self.assertEqual( d, IECore.CompoundData() )
 
-		p.applyTweaks( d, missingMode = GafferScene.TweakPlug.MissingMode.Ignore )
+		self.assertFalse( p.applyTweaks( d, missingMode = GafferScene.TweakPlug.MissingMode.Ignore ) )
```

Comment:
All our `assertFalse()` cases seem to stem from `Ignore` mode. Would be good to have a test dedicated to false return due to all tweaks disabled or with empty names.

Reply:
Good call, added more tests in https://github.com/GafferHQ/gaffer/pull/3699/commits/4c8dfa822777076457e4d59a6524aef177566d2b.